opts/options.go

```diff
@@ -18,13 +19,22 @@ var (
 	DefaultProtocols             = []protocol.ID{ProtocolDHT}
 )
 
+// BootstrapConfig specifies parameters used for bootstrapping the DHT.
+type BootstrapConfig struct {
+	BucketPeriod             time.Duration // how long to wait for a k-bucket to be queried before doing a random walk on it
+	Timeout                  time.Duration // how long to wait for a bootstrap query to run
+	RoutingTableScanInterval time.Duration // how often to scan the RT for k-buckets that haven't been queried since the given period
```

Comment:
Why don't we refresh every `BucketPeriod`?

Reply:
Great question. I can not think of why. @Stebalien, would you happen to remember why we agreed on a `RoutingTableScanInterval` of 30 minutes here ?

Reply:
I can't remember. I believe what we really want is for the scan interval to be some fraction of the bucket period. I'd be fine getting rid of this.

Reply:
Really, we technically don't need a scan interval. We could just take the minimum of all bucket update times and add the bucket period to that. That would have the added benefit of spreading bucket polls out over time instead of batching them together.